src/modules/rlm_rest/rest.c

```diff
@@ -1619,6 +1620,14 @@ static size_t rest_response_header(void *in, size_t size, size_t nmemb, void *us
 
 		ctx->code = atoi(p);
 
+		/*
+		 *  Save the HTTP return status code.
+		 */
+		vp = pair_make_reply("REST-HTTP-Status-Code", NULL, T_OP_SET);
+		vp->vp_integer = ctx->code;
+
+		RDEBUG2("Adding reply:REST-HTTP-Status-Code += \"%d\"", vp->vp_integer);
```

Comment:
Better to handle this in `rest_request_perform`.  You can get the ctx from the curl handle.